src/client/app/+about/components/about.component.spec.ts

```diff
@@ -2,25 +2,29 @@ import {
   TestComponentBuilder,
   describe,
   expect,
-  injectAsync,
-  it
-} from 'angular2/testing';
+  inject,
+  it,
+  AsyncTestCompleter
+} from 'angular2/testing_internal';
```

Comment:
It should use `testing` module as `testing_internal` is used to test the framework itself.